aws/data_source_aws_elb_hosted_zone_id.go

```diff
@@ -26,7 +28,6 @@ var elbHostedZoneIdPerRegionMap = map[string]string{
 	"sa-east-1":      "Z2P70J7HTTTPLU",
 	"us-east-1":      "Z35SXDOTRQ7X7K",
 	"us-east-2":      "Z3AADJGX6KTTL2",
-	"us-gov-west-1":  "048591011584",
```

Comment:
This is not a Route 53 hosted zone ID, it's the [AWS account ID for access log delivery](https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/govcloud-elb.html).
See `aws/data_source_aws_elb_service_account.go`.